packages/@react-spectrum/utils/package.json

```diff
@@ -27,6 +27,7 @@
     "clsx": "^1.1.1"
   },
   "peerDependencies": {
+    "@react-spectrum/provider": "^3.1.2",
```

Comment:
I think you can remove this now

Reply:
https://github.com/adobe/react-spectrum/pull/1684#discussion_r593526577